src/index.js

```diff
@@ -63,3 +64,7 @@ export default (options: options = {}): Promise<any> => {
   });
 
 };
+
+animateValue.easings = Object.assign({}, namedEasings);
+
```

Comment:
That could be a good idea, i'll have to investigate Lerna as I haven't used it before. Should I leave as is for this PR?